lib/scss_lint/linter/single_line_per_property.rb

```diff
@@ -4,6 +4,8 @@ class Linter::SingleLinePerProperty < Linter
     include LinterRegistry
 
     def visit_rule(node)
+      visit_children(node)
```

Comment:
Ah, I guess that's why my grepping for `visit_children` didn't really turn up much.